broker.go

```diff
@@ -51,7 +51,15 @@ type Broker struct {
 	brokerRequestsInFlight metrics.Counter
 	brokerThrottleTime     metrics.Histogram
 
-	kerberosAuthenticator GSSAPIKerberosAuth
+	kerberosAuthenticator          GSSAPIKerberosAuth
+	connLockWaitTime               metrics.Histogram
+	brokerConnLockWaitTime         metrics.Histogram
```

Comment:
It seems like these pairs track the same metric, for my education, why do we need both?

Reply:
The name of the metric for broker is changed by appending "-for-broker" at the end of the metric name when we register it.